### ISSUE TITLE
Registry prop for tracking forked repos.

### ISSUE THREAD

**Issue by dadambickford** [npm/registry-issue-archive#28]
After forking a repo recently and updating the package.json within, I felt inauthentic replacing the author prop with my own name. Sure it was my repo now but I hadn't really changed anything yet, so the vast majority of code was not mine. I asked my partner @oscargodson if he had heard of any kind of 'fork-author' prop, that I could add to respect the original author's work. We couldn't find anything and @rockbot suggested recommending here.

Basically the idea would be some sort of 'fork-author' prop, that would be added instead of replacing the 'author' prop. That way you could link back to the parent repo, and potentially further down the line track the whole tree of forks. It could be a nice way for package.json to reflect all of the past and present authors of a project.

**Comment by terinjokes** [comment]
If you don't feel comfortable updating `author`, you can instead add yourself to the `contributors` array.  The [people](https://docs.npmjs.com/files/package.json#people-fields-author-contributors) in the contributors array can be an object, and nothing stops you from adding additional keys.

**Comment by ashleygwilliams** [comment]
i've run into this exact problem and def feel that sense of disingenuousness in listing myself as author. the `contributors` array seems like the way to go here! am going to close, but please reopen if you still have questions. thanks!

**Comment by OscarGodson** [comment]
Contributors is nice, but it doesn't really solve what I was looking for. The idea was to link back to the parent module.

Just as an example, iojs would link back to nodejs in the "forks" sidebar section. If there was a fork of iojs, that fork would link to iojs which then links to nodejs. That way you can always find the originating module. Like how GitHub works with forks. Some forks are vastly different from the originator so it's not easy to get to the original source unless you read licenses and stuff.
